<ui version="4.0" >
 <class>AddAttributeWizardDesign</class>
 <widget class="Q3Wizard" name="AddAttributeWizardDesign" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>505</height>
   </rect>
  </property>
  <property name="caption" stdset="0" >
   <string>Add Attribute</string>
  </property>
  <widget class="QWidget" name="WizardPage" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="imageLabel" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="text" >
       <string comment="DO NOT TRANSLATE" >PI</string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QComboBox" name="attributeBox" />
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="textLabel4" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text" >
       <string>Attribute:</string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2" >
     <widget class="Line" name="line3" >
      <property name="frameShape" >
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
    <item rowspan="2" row="0" column="1" >
     <widget class="QLabel" name="textLabel2" >
      <property name="text" >
       <string>&lt;p>Please select an attribute you want to add to the current entry.&lt;/p>
&lt;p>By default only attributes which are supported by the current 
objectclasses are displayed. If you want to add an attribute which is 
supported by other objectclasses, please enable this functionality below.
&lt;/p></string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2" >
     <widget class="QCheckBox" name="binaryBox" >
      <property name="text" >
       <string>Use binary extension</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>21</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="2" >
     <widget class="QCheckBox" name="enableAllBox" >
      <property name="text" >
       <string>Enable all attributes which are supported by the server.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="WizardPage" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="2" column="0" colspan="2" >
     <widget class="Line" name="line4" >
      <property name="frameShape" >
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="0" column="1" >
     <widget class="QLabel" name="textLabel2_2" >
      <property name="text" >
       <string>&lt;p>You have chosen to add an attribute which is not supported by the 
objectclasses for the current entry.&lt;/p>
&lt;p>Please select an objectclass which supports the new attribute. The 
list on the right shows all attributes which must be added additionally with
the selected objectclass.&lt;/p></string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2" >
     <widget class="QSplitter" name="splitter1" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible" >
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layout1" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel3_2" >
          <property name="text" >
           <string>&lt;b>Objectclass&lt;/b></string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Q3ListBox" name="classBox" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layout2" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel4_2" >
          <property name="text" >
           <string>&lt;b>Additional attributes&lt;/b></string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Q3ListBox" name="mustAttributeBox" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="objectclassLabel" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="text" >
       <string>CL</string>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>21</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Wizard</class>
   <extends>QDialog</extends>
   <header>q3wizard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>enableAllBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>AddAttributeWizardDesign</receiver>
   <slot>initAttributeBox()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>attributeBox</sender>
   <signal>activated(QString)</signal>
   <receiver>AddAttributeWizardDesign</receiver>
   <slot>newSelection()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>classBox</sender>
   <signal>selectionChanged()</signal>
   <receiver>AddAttributeWizardDesign</receiver>
   <slot>classSelection()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
